contracts/ConstAddressDeployer.sol

```diff
@@ -52,4 +79,18 @@ contract ConstAddressDeployer {
             )
         );
     }
+
+    function _deploy(bytes memory bytecode, address sender, bytes32 salt) internal returns (address deployedAddress_) {
+        if (bytecode.length == 0) revert EmptyBytecode();
+
+        bytes32 newSalt = keccak256(abi.encode(sender, salt));
```

Comment:
```suggestion
```

Reply:
I would like to include the original `salt` in the emitted event below and reduce duplication.

Reply:
but why? The original salt can be found in the tx data, which is even easier to be located than something in the event.

Reply:
Easier to spot it in events in the explorer, but I'm ok with showing the derived salt too.